types/phplint/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for phplint 2.0
+// Project: https://github.com/wayneashleyberry/phplint
+// Definitions by: Adam Thompson-Sharpe <https://github.com/MysteryBlokHed>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="grunt" />
+
+export interface Options {
+    stdout?: boolean | undefined;
+    stderr?: boolean | undefined;
```

Comment:
those are not booleans, but `stdout` or `stderr` instances, e.g. if someone wants to hook into output from this module. so `opts.stdout` is expression evaluation, not boolean check afaik

Reply:
It looks like the only time `stdout` or `stderr` are used is in if statements like this:

https://github.com/wayneashleyberry/phplint/blob/e653ec32a94b02f3db0128e0d568d6afd59db09b/index.js#L48-L58

https://github.com/wayneashleyberry/phplint/blob/e653ec32a94b02f3db0128e0d568d6afd59db09b/index.js#L83-L84

If it's true, a different variable is written to stdout/stderr

Reply:
here is the definition of the write method
```
write(buffer: string | Uint8Array, cb?: ((err?: Error | undefined) => void) | undefined): boolean
```
so `stdout` and `stderr` are type of `string | Uint8Array`. I think so.

Reply:
I think that `stdout` and `stderr` on the Options interface are booleans for whether to log to stdout and stderr or not.
The only time I think that data to print to stdout/stderr is actually passed is in the [callback type](https://github.com/DefinitelyTyped/DefinitelyTyped/pull/58379/files/ee2fc302c0504ee93bcde1d15ca6970ba42c78c3#diff-51e68319df901acfde59e66c16aa00df0fc7fc21eefd603b94b0c886b5d442d8R18).
I should probably change the callback's stdout/stderr parameter types to `string | Uint8Array`, but I don't think the keys on the Options interface need to be changed.

Reply:
@MysteryBlokHed you're correct and sorry for posting wrong suggestions (I was thinking about other module pattern), the readme describes those as booleans:
https://github.com/wayneashleyberry/phplint/blob/e653ec32a94b02f3db0128e0d568d6afd59db09b/README.md